ArduCopter/mode.cpp

```diff
@@ -560,6 +560,48 @@ void Copter::Mode::land_run_horizontal_control()
     attitude_control->input_euler_angle_roll_pitch_euler_rate_yaw(nav_roll, nav_pitch, target_yaw_rate);
 }
 
+float Copter::Mode::throttle_hover() const
+{
+    return motors->get_throttle_hover();
```

Comment:
This whole wrapper exists only to call here directly, and it only gets called once. Can this just be inlined?

Reply:
This method is overridden by ModeAcro so users can specify their acro throttle.  Bizarre, but there you go.